### DISCUSSION TITLE
What Resources do You Guys Use to Download Content?

### DISCUSSION THREAD

**Discussion by Bluemoondragon07** [arianneorpilla/jidoujisho#278] [Show and tell]
Hi, I was wondering what resources everyone uses to download content to use with Jidoujisho. 

I like to download books from [Z-Library](https://z-lib.is). Their mobile app is also great. I am currently reading 鹿の王 (Deer King), and though it's kind of hard to understand (they are incubating a shining-marijuana egg in the beginning or something?), it's also great.

I haven't found a place to get content for Mokuro yet, and I'm not quite sure how it works yet. Nor have I really found a good source for the video player.

Right now, I just downloaded すずめの戸締まりfrom an Android movie-downloader app that uses torrents. Haven't watched it yet, kind of confused about the subtitles. It can't auto generate subtitles, sadly, it looks like. Downloading with that method takes waaay too long, so I recommend maybe just downloading an MP4 from BiliBili. I could have done that, but I was scared to make an account for some reason.

I'll put more links in the comments below once I find them.

**Comment by Bluemoondragon07** [comment]
[Here](https://drive.google.com/drive/folders/1tTdLppnqMfVC5otPlX_cs4ixlIgjv_lH) is a Google Drive with several Yomichan dictionaries. Most work, but some don't. I recommend JMDict Extra, Kireicake, Kanjidic, and at least 1 monolingual dictionary. I also like this one, can't remember the whole name, but it has ハイブリッド in it somewhere.

**Reply by arianneorpilla** [reply]
You shouldn't use JMdict Extra, if you do you'll notice some entries are missing after being imported. If a dictionary shows that it has structured content, you shouldn't use it and use a version that doesn't have those. There are other versions of JMdict that are just plain text.

**Comment by Bluemoondragon07** [comment]
As for fonts, I love reading in ShinseiKai Pro, HanaMinA, and Klee One Semibold. Klee One Seminole is included in the app. Sometimes I like Seto, sometimes I hate it. Depends on what I'm reading. Han Serif is also good. You can download the fonts I mentioned [here](https://disk.yandex.com/d/cj0hetiqGjhDiA).

**Comment by arianneorpilla** [comment]
I would just get videos from Nyaa and subtitles from Kitsunekko. https://mokuro.moe for manga content (you can add it as a catalog) and just watch subtitled content from YouTube, can long press on a video to go to its channel.

**Comment by brishtibheja** [comment]
@Bluemoondragon07 I download videos from telegram. It's soft subbed almost all of the time. There's an issue with privacy but my government doesn't care so depends on where you live. I used to use books from itazuraneko but I've heard yesterday that it was taken down. For manga, you can run mokuro on them by your own. You will have to download them from something like manga-zip/dl-zip/raw-zip. Aside from that, You can read web novels from syosetsu. For offline reading you will just have to convert them into epub using something like WEBtoEpub. Look into the resource page of the moe way for more information.

**Reply by Bluemoondragon07** [reply]
Thanks for the resources. I will look into Telegram! I just looked at mokuro and I will experiment with it.

**Reply by brishtibheja** [reply]
I'll just add one more thing in case you don't know about it yet. Anna's archive is an archive of books from different shadow libraries. To download japanese books search with japanese names in stead of English ones.

Edit: Do you know about jpdb? You can find there difficulty ratings for almost any anime/ln/manga. learn natively is also great for the same purpose.

**Reply by Bluemoondragon07** [reply]
Just checked out Anna's archive and it's great; thanks! It might even be better than Z-Libary. I also did not know about jpdb and learn natively. I looked at learn natively and wooow! I will definitely use it when I look for books and movies. どもありがとう！
